### ISSUE TITLE
Add a restore timeout

### ISSUE THREAD

**Issue by huksys** [PFCKrutonium/Windows-10-Login-Background-Changer#206]
As some users may run into an unusable login screen after trying to apply an image, an automatic restore of the back-upped PRI file after a specific timeout, like these ones when you change the display resolution (see screenshot), could come quite handy.

![unbenannt](https://cloud.githubusercontent.com/assets/8116260/9291321/ccee4256-43b8-11e5-86cb-d3312ceae51c.png)

**Comment by xieofxie** [comment]
Good idea. Also I find that when I use a smaller size of the same jpg image by lowering quality, it will not break out. Maybe it works just as win7 which requires an image file < 255kb?
